src/dotnet/Notification.UI.Blazor/NotificationUI.cs

```diff
@@ -67,9 +69,10 @@ public async Task EnsureDeviceRegistered(CancellationToken cancellationToken)
             if (_deviceId.Value != null)
                 return;
 
-        var deviceId = await DeviceTokenRetriever.GetDeviceToken(cancellationToken);
-        if (deviceId != null)
-            _ = RegisterDevice(deviceId, cancellationToken);
+        var deviceId = await DeviceTokenRetriever.GetDeviceToken(cancellationToken).ConfigureAwait(false);
+        if (deviceId != null) {
```

Comment:
We don't use brackets for single line statement

Reply:
fixed